packages/core/src/modules/employment/use-cases/send-company-review-notifications.ts

```diff
@@ -0,0 +1,63 @@
+import dedent from 'dedent';
+
+import { db } from '@oyster/db';
+
+import { job } from '@/infrastructure/bull/use-cases/job';
+import { ENV } from '@/shared/env';
+
+/**
+ * This is a weekly job that runs and finds all students that have had a work
+ * experience end in the last week. Then, it has the ColorStack bot send a DM
+ * to all these students suggesting that they add a review of their experience.
+ */
```

Comment:
Let's update the function spec to reflect the change to making it a monthly job rather than weekly